.github/workflows/ci.yml

```diff
@@ -17,6 +17,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@1.0.4
```

Comment:
Looks like this should be `@v1`: https://github.com/marketplace/actions/gradle-wrapper-validation

Reply:
fixed